scripts/ctp7_registers.py

```diff
@@ -0,0 +1,78 @@
+__author__ = 'evka'
+
+import xml.etree.ElementTree as xml
+import textwrap as tw
+
+ADDRESS_TABLE_TOP = '../doc/address_table/gem_amc_top.xml'
```

Comment:
while I'm not 100% sure, I believe this script also should not reside on the ctp7 at is actually used on the host PC. Evaldas should confirm. However I'm pretty sure it is NOT used to load the firmware neither it is coupled to sca or reg_interface tools